Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOAddressSummary.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import SwiftUI
+
+struct PPOAddressSummary: View {
+  let address: String
+
+  var body: some View {
+    HStack(alignment: .firstTextBaseline) {
+      // TODO: Localize
+      Text("Shipping address")
+        .lineLimit(nil)
```

Comment:
We do not need `.lineLimit(nil)`, as default `lineLimit` for text component is `nil`